Sources/BraveWallet/Settings/Web3SettingsView.swift

```diff
@@ -48,12 +51,40 @@ public struct Web3SettingsView: View {
       }
       if let ipfsAPI { // means users come from the browser not the wallet
         Section(
-          header: Text(Strings.Wallet.ipfsSettingsHeader),
-          footer: Text(Strings.Wallet.ipfsSettingsFooter)
+          header: Text(Strings.Wallet.ipfsSettingsHeader)
         ) {
+          Picker(selection: $enableIPFSResourcesResolver.value) {
+            ForEach(Preferences.Wallet.Web3IPFSOption.allCases) { option in
+              Text(option.name)
+                .foregroundColor(Color(.secondaryBraveLabel))
+                .tag(option)
+            }
+          } label: {
+            VStack(alignment: .leading, spacing: 6) {
+              Text(Strings.Wallet.ipfsResourcesOptionsTitle)
+                .foregroundColor(Color(.braveLabel))
+                .padding(.vertical, 5)
```

Comment:
Need to adjust the spacing in a couple spots in this file to be multiple of 2. Ideally this row should match how `Allow ENS Offchain Lookup` row looks other than copy 🙂

![Simulator Screen Shot - iPhone 12 Pro - 2023-03-24 at 15 50 31](https://user-images.githubusercontent.com/5314553/227625932-31ab1c01-b5c7-4f8d-8567-1bbe836b23f4.png)